examples/helpers/android/app/build.gradle.in

```diff
@@ -50,6 +50,7 @@ android {
     externalNativeBuild {
         cmake {
             path "CMakeLists.txt"
+            version "3.10.2"
```

Comment:
```suggestion
```
I would consider to use default CMake version (3.6) which comes with SDK manager. See previous comment